hermes-management/src/main/java/pl/allegro/tech/hermes/management/infrastructure/metrics/HybridSubscriptionMetricsRepository.java

```diff
@@ -39,22 +37,22 @@
 
     private final MetricsPaths metricsPaths;
 
-    private final SharedCounter sharedCounter;
+    private final SummedSharedCounter summedSharedCounter;
 
-    private final DistributedEphemeralCounter distributedCounter;
+    private final SummedDistributedEphemeralCounter summedDistributedCounter;
 
     private final ZookeeperPaths zookeeperPaths;
 
     private final SubscriptionLagSource lagSource;
 
     @Autowired
```

Comment:
I think that `@Autowired` is not necessary here.

Reply:
Done